code/modules/mob/living/silicon/robot/robot_mob.dm

```diff
@@ -1420,6 +1456,14 @@ GLOBAL_LIST_INIT(robot_verbs_default, list(
 	else
 		REMOVE_TRAITS_IN(src, LOCKDOWN_TRAIT)
 
+// Allows the borg to unlock themselves after a lengthy period of time.
+/mob/living/silicon/robot/proc/lockdown_override()
+	SetLockdown(FALSE)
+	log_admin("<span class='notice'>The lockdown on cyborg [src] has been lifted because their lockdown timer expired.</span>")
```

Comment:
Don't use HTML in log files. Make this message_admins or remove the HTML.

Reply:
HTML removed.